### ISSUE TITLE
PDO Firebird $_transactionID property

### ISSUE THREAD

**Issue by dregad** [ADOdb/ADOdb#935]
While going through #926, I reviewed usage of the `_transactionID` property within PDO Firebird driver.

It is only accessed in *_affectedrows()* method when calling *fbird_affected_rows()*, but never actually set.

The native Firebird driver initializes it from fbird_trans()'s return value, when initializing the transaction in beginTrans().

AFAICT, it is not possible  to retrieve the transaction ID using PDO functions - PDO::beginTransaction() just returns a boolean, so I believe the PDO driver should be modified like this.

```diff
	public function _affectedrows()
	{
-		return fbird_affected_rows($this->_transactionID ? $this->_transactionID : $this->_connectionID);
+		return fbird_affected_rows($this->_connectionID);
	}
```

 @mnewnham thoughts ?

**Comment by mnewnham** [comment]
Surely this entire function is a copy/paste mistake because it's calling native mode functionality, _**fbird_affected_rows()**_ from within the pdo driver and that function wouldn't exist. The entire function should be removed. (it doesnt exist in any other driver apart from the base class).

**Comment by dregad** [comment]
OK thanks I'll do that

**Comment by dregad** [comment]
Fixed in 5766f6b17aac1da80302840416e24ac9a341742c
